src/Components/Header/Header.tsx

```diff
@@ -0,0 +1,29 @@
+import logoImage from "../../images/XriiLogo/gradient logo.svg";
+import { Link } from "react-router-dom";
+import paths from "../../Paths/paths";
```

Comment:
you can use alias @/ also here

Reply:
Fixed to import paths from '@paths/paths';